AZ3166/src/cores/arduino/AttachInterrupt.cpp

```diff
@@ -0,0 +1,30 @@
+#include "AttachInterrupt.h"
+#include "wiring_constants.h"
+
+static PinName validPinames[16] = {PB_0, NC, PB_2, PB_3, PA_4, PA_5, PB_6, PB_7, PB_8, PB_9, PA_10, NC, NC, PB_13, PB_14, PB_15};
```

Comment:
Comment all the valid Pin name in the header file and provide more descriptions

Reply:
Done.